app/views/dashboard/callouts/show.html.haml

```diff
@@ -55,7 +55,7 @@
           = Callout.human_attribute_name(:commune_ids)
         %dd.col-sm-7.col-md-6
           %code
-            = location_names(resource.commune_ids, Pumi::Commune)
+            = location_names(resource.commune_ids, Pumi::Commune, I18n.locale)
```

Comment:
call this with `curent_user.locale` if for some reason `I18n.locale` returns nil